pages/live/artisten-halle-2019.rst

```diff
@@ -35,3 +40,13 @@ Cast
 * Paul Schuladen (Tech, Recording)
 * Robin Burke (Modular Synthesizer)
 * Rodrigo Sena (Modular Synthesizer)
+
+
+
+.. image:: /images/20190308-eoc-flyer.jpg
```

Comment:
We could also use the minimized version of this picture (e.g. 20190308-eoc-flyer.thumbnail.jpg).
Doesn't have to be now though :)